networking/proxies/configuring-cluster-wide-proxy.adoc

```diff
@@ -0,0 +1,32 @@
+include::modules/common-attributes.adoc[]
+ifdef::openshift-dedicated[]
+include::modules/attributes-openshift-dedicated.adoc[]
+endif::[]
+:context: cluster-wide-proxy-configuration
+[id="cluster-wide-proxy-configuration"]
+= Configuring a cluster-wide proxy during installation
+
+toc::[]
+
+Cluster-wide proxies may be configured during cluster creation.
+
+.Prerequisites
```

Comment:
If you're going to have a prerequisites section in an assembly instead of a module, you should use the == Prerequisites heading type. For example:

```
[id="prerequisites_configuring-alert-notifications"]
== Prerequisites
```
(but with the right context in the ID of course). Details here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies